h5p-bildetema/src/components/SearchPage/SearchPage.tsx

```diff
@@ -36,12 +37,11 @@ const SearchParamKeys = {
 const SearchPage = (): JSX.Element => {
   const languages = useLanguagesWithTranslatedLabels();
   const langCode = useCurrentLanguageCode();
+  const siteLanguage = useSiteLanguage();
 
   const [searchParams, setSearchParams] = useSearchParams();
   const [searchLanguage, setSearchLanguage] = React.useState<Language>(
-    languages.find(lang => lang.code === langCode) ||
-      // TODO should not be static
-      ({ code: langCode, label: "Bokmål" } as Language),
+    languages.find(lang => lang.code === langCode) || siteLanguage,
```

Comment:
Very good change! ({ code: langCode, label: "Bokmål" } as Language) could have been an issue

Reply:
Yes, it should be fine